GWAS/PCA.ipynb

```diff
@@ -1153,13 +1151,8 @@
     "    # Make the plots\n",
     "    ###\n",
     "    # Get the min and max values for x and y-axes\n",
-    "    if (${\"T\" if len(min_axis) == 0 or len(max_axis) == 0 else \"F\"}) {\n",
-    "        min_axis <- round(colMins(as.matrix(f$projection[sapply(f$projection, is.numeric)])),1)\n",
-    "        max_axis <- round(colMaxs(as.matrix(f$projection[sapply(f$projection, is.numeric)])),1)  \n",
-    "    } else {\n",
-    "        min_axis <- ${min_axis}\n",
-    "        max_axis <- ${max_axis}\n",
-    "    }\n",
+    "    min_axis <- round(colMins(as.matrix(f$projection[sapply(f$projection, is.numeric)])),1)\n",
```

Comment:
@UxxUnet why are we doing this? sometimes we do want to be able to specify min and max axies and not let that depend on data

Reply:
By default, min_axis and max_axis are not specified, right? they are default to empty `""`

Reply:
What we want is that when the min_axis and max_axis are set to 0 or none the code defaults to the firs part
```
 if (${"TRUE" if len(min_axis) == 0 or len(max_axis) == 0 else "FALSE"}) {
        min_axis <- round(colMins(as.matrix(f$projection[sapply(f$projection, is.numeric)])),1)
        max_axis <- round(colMaxs(as.matrix(f$projection[sapply(f$projection, is.numeric)])),1)  
    } else {
        min_axis <- ${min_axis}
        max_axis <- ${max_axis}
```
That behavior is currently not happening. And that's what we need to fix